go.mod

```diff
@@ -5,20 +5,20 @@ go 1.22
 require (
 	github.com/akutz/gosync v0.1.0
 	github.com/apparentlymart/go-cidr v1.1.0
-	github.com/container-storage-interface/spec v1.6.0
+	github.com/container-storage-interface/spec v1.10.0
```

Comment:
@delldubey We are using CSI Spec v1.6.0, this change should be reverted.

Reply:
It is reverted, one of the dependencies updated that.